PIM/Operation/04_ManageView.md

```diff
@@ -0,0 +1,179 @@
+[!!DataHub](../../DataHub/Overview/01_General.md)
+[!!User Interface Product list](../UserInterface/02a_List.md)   
+[!!Create a language](../../DataHub/Integration/CreateLanguage.md)
+[!!Configure the language settings](../Integration/04_ConfigureLanguages.md)
+[!!Create a channel](../../DataHub/Integration/CreateChannel.md)
+[!!Configure the channel settings](../Integration/05_ConfigureChannels.md)
+[!!Manage the catalogs](../Integration/06_ManageCatalogs.md)
+
+# Manage the view
+
+The *LIST* tab is the view where you can see and manage the products. To have a customized view for all purposes, you can modify various settings to adjust the view to your requirements and save individual views to quickly access the view you need.
+
+
+## Select a language
+
+You can select the language for which the attribute values of the products are displayed. Only multi-language attributes can have different values in different languages. Otherwise, the value remain the same in all languages. Note that the selected language only refers to the display language of the attribute values but does not equal to the user interface language. To change the interface language, you have to [change the user language settings](to_be_completed).
```

Comment:
```suggestion
You can select the language for which the attribute values of the products are displayed. Only multi-language attributes can have different values in different languages. Otherwise, the value remain the same in all languages. Note that the selected language only refers to the display language of the attribute values but does not equal to the user interface language. To change the interface language, you have to change the user language settings.
```
[comment]: <> (Link hinzufügen, wenn verfügbar)